Makefile

```diff
@@ -96,6 +104,17 @@ push-public: clean build-debian
 		docker push ${REPOSITORY}/cp-$${component}:${CP_VERSION} || exit 1; \
   done
 
+push-nexus:
+	for component in ${COMPONENTS} ; do \
+		echo "\n Pushing cp-$${component}  \n==========================================\n "; \
+		docker tag ${REPOSITORY}/cp-$${component}:latest docker.confluent.io:5000/${REPOSITORY}/cp-$${component}:latest || exit 1; \
+		docker tag ${REPOSITORY}/cp-$${component}:${VERSION} docker.confluent.io:5000/${REPOSITORY}/cp-$${component}:${VERSION} || exit 1; \
+		docker tag ${REPOSITORY}/cp-$${component}:${CP_VERSION} docker.confluent.io:5000/${REPOSITORY}/cp-$${component}:${CP_VERSION} || exit 1; \
+		docker push docker.confluent.io:5000/${REPOSITORY}/cp-$${component}:latest || exit 1; \
+		docker push docker.confluent.io:5000/${REPOSITORY}/cp-$${component}:${VERSION} || exit 1; \
+		docker push docker.confluent.io:5000/${REPOSITORY}/cp-$${component}:${CP_VERSION} || exit 1; \
```

Comment:
👍  to just making it a variable. That's also nice for anyone that may be customizing the images themselves and want to publish to their own repo.